### ISSUE TITLE
Plural of "they" is "he". Bug it becomes "she".

### ISSUE THREAD

**Issue by Noitidart** [blakeembrey/pluralize#154]
The plural of "they" is "he". It is becoming "she". This was an embarrassing when my community pointed out this mistake with feedback like "are you giving us all love?". Please fix to respect giving "he".

"She" should be for "country" "boats" or other inanimate objects. This is hard to detect but "they" is always used humans, so respect giving "he" is accurate.

**Comment by blakeembrey** [comment]
The plural of "they" is "they", not "he". But I think it's better to keep pronouns out of scope for this project in the long term. Pluralizing is clear, but singularizing it back is not.

**Comment by Noitidart** [comment]
I agree it's not clear going in reverse. However "he" doesn't carry the love that "she" does. Would the fix be swapping the lines here:

https://github.com/plurals/pluralize/blob/master/pluralize.js#L275

I would love to provide a PR if this is the case.

Thanks again for your prompt replies!

**Comment by blakeembrey** [comment]
Why wouldn’t it be the singular sexless they? The bug to me is that singularization here did anything at all.

**Comment by Noitidart** [comment]
I don't think there is sex, it's respect and love. Like he in regards to a person and she in regards to country. They does not convey that respect.

**Comment by blakeembrey** [comment]
Do you have a reference for this? I’ve never heard of it.

**Comment by Noitidart** [comment]
Hi brother my reference is everyone in my experience, in multiple languages. The most recent is the embarrassment I encountered when multiple unrelated users reported this bug to me. So maybe it should be "they" (i respectfully disagree based on my experience) or "he", but "she" is not an option for general respect in any language, I knocked up the PR I hope you accept please - https://github.com/plurals/pluralize/pull/156

**Comment by joeyfigaro** [comment]
@Noitidart There may be a bit of a language barrier here? It's unclear what exactly `I don't think there is sex, it's respect and love. Like he in regards to a person and she in regards to country.` means.

**Comment by Noitidart** [comment]
Hi Joey, thanks for trying to understand. It's not a lang barrier, I think it's a political one. No problem. If author wants to leave "they" going to "she" instead of the correct "he" up to him. I've explained as best as I can. I have worked around the issue in my app, was trying to help others.

**Comment by BrenoVelasco** [comment]
What did I just read??
@Noitidart I know this is "fixed" but could you provide the actual sentence that you used? I'm really curious.

**Comment by BrenoVelasco** [comment]
Thanks a lot for your reply, @Noitidart. I can now better understand this issue.
I'm not sure about the real implications regarding using "she" instead of "he" in this sentence (maybe because I'm Brazilian and we have different cultural implications, hehe).
See if something like this wouldn't help solve your issue:

**There is one pending guest. If the guest shows up please allow entry.**
**There are 2 pending guests. If they show up please allow entry.**

**Comment by Noitidart** [comment]
Thanks Breno no problem. I preferred the "he" as it gives a sense of respect and translates well in other languages.

**Comment by evenfrost** [comment]
> There is one pending guest, if they show up please allow entry.

is the only correct form if a subject's gender is unknown.

I'm not a native English speaker, but even I know it. Genuinely surprised this thread even exists.

**Comment by Noitidart** [comment]
Interesting replies. I grew up in the 90s, shows like "Saved by the Bell" and everything else used "he". 

Might be a new generation in the "west" and wherever these ideologies affected. Countries where a majority aren't affected by the "west", like African, Asian, South American, South East Asian, Mid-east, Russian all use "he" till today, here's a recent blog - https://thoughtspile.github.io/2021/10/18/non-react-state/

None the less, even according to your rules, turning plural "they" to singular "she" is wrong. But I get it, and it's ok, it's not my lib. I think the lib authors here are western ideology so prefer "she".

**Comment by vixalien** [comment]
Dead issue but the singular of "they" when no context is provided should be "they" for gender neutrality. Example

> The girls were hungry, they went to KFC

Singular

> The girl was hungry, they went to KFC

Example 2

> The wolf attacked the boys. They chased it.

Singular

> The wolf attacked the boy. They chased it.

**Comment by mcelligottnick** [comment]
@Noitidart  its not about prefering 'he' or 'she'. It's about keep the sentence accurate with the information supplied. 'they' has no gender, so the singular needs to be neutral as well, which is 'they' (simiar with them). 

@vixalien I think you may have confused yourself, because both examples you are given the gender of those in the sentence. But they are also slightly confusing english sentences because of the indirect object. Here are some correct examples, including pointing out the errors in yours;

**Subjects have gender;**

Plural: The girls were hungry, they went to Lord Of The Fries.
Singular: The girl was hungry, she went to Lord Of The Fries.

Plural: The wolf attacked the boys. They chased it.
Singular: The wolf attacked the boy. He chased it.

**No gender**

Plural: The people were hungry, they went to Lord Of The Fries.
Singular: The person was hungry, **they** went to Lord Of The Fries.

Plural: The people were shocked, it really affected them.
Singular: The person was shocked, it really affected **them**.
